### ISSUE TITLE
Vimtex warning message for bbl file 

### ISSUE THREAD

**Issue by zhuosongz** [lervag/vimtex#1401]
### Issue description 

Please find my `minivimrc` and `main.tex` from: 

https://github.com/zhuosongz/vimtexTest/tree/master/vimtexTest

When I run `mvim -u minivimrc main.tex`, and compile the `tex` file (using `\ll`), and it appears some warning message using `\le`: 

```
main.tex|20 warning| Underfull \hbox (badness 2126) in paragraph at lines 20--23
```
When I press `<Enter>` on the warning message, it should jump to line 20--23 of `main.bbl` file, *but it jumped to the line 20--23 of the `main.tex` file*, which is the wrong target file. 

Could you please help me on this issue? Thank you very much!

**Comment by lervag** [comment]
Thanks. This is a known issue that is very hard to fix. The problem is that the error output in the log file is difficult to parse. For the present example, it might look quite easy, i.e.

```
 (./main.bbl
Underfull \hbox (badness 2828) in paragraph at lines 20--23
[]\OT1/cmr/m/n/10 Chen, W.-K. (2015). \OT1/cmr/m/it/10 Vari-a-tional rep-re-sen-ta-tions for the Parisi func-
 []
```

Here it seems easy to capture the `main.bbl` part. The problem is that we have to do this with the `:help errorformat` system in Vim. I've managed to do this partly by using the `%P` and `%Q` directives, **but**: In many cases, the filenames that are provided are so long that they are broken into multiple lines. This breaks the errorformat and the entire parser. In the end, I had two options: To make the parser properly push new files on the stack and support cases like the present case, which would often fail totally for many projects/cases, or to fall back to the current behaviour where some cases (like the present) will not be properly parsed.

Long story short: I'm sorry to say that I have no idea how to fix this in a reliable manner.

**However**: vimtex does support other methods of parsing the errors. I've tested and found that `pplatex` will properly detect this kind of error. If you install it, then set `let g:vimtex_quickfix_method = "pplatex"`, then things should look better.
